setup.py

```diff
@@ -77,8 +78,10 @@
     package_dir      = {
         'url': 'url'
     },
+    package_data     = {
+        'url': ['psl/*']
+    },
     install_requires = [
-        'publicsuffix'
```

Comment:
Do we need to keep an empty install_requires list, or lose the while thing until we need it again?